src/codecup.css

```diff
@@ -25,7 +25,7 @@
 	padding: 5px;
 	padding-left: 10px;
 	padding-right: 10px;
-	right: 0;
+	right: 0px; 
```

Comment:
If `right: 0px` is used, the language display name may not be shown properly when a scrollbar is present.
In my PR, I have adjusted this to `right: 12px` to ensure proper visibility.

<img width="701" alt="image" src="https://github.com/user-attachments/assets/5f6f2bb3-de07-4f10-b3b3-6574f7e93d9c" />